packages/@aws-cdk/aws-stepfunctions-tasks/test/eventbridge/put-events.integ.snapshot/asset.41fc8f2dc7c01b34dda9916c7f763e7b7909eb629da9ffe879cb786114aae736.bundle/index.js

```diff
@@ -509,27 +512,9 @@ var AssertionHandler = class extends CustomResourceHandler {
 };
 var MatchCreator = class {
   constructor(obj) {
-    switch (Object.keys(obj)[0]) {
-      case "$ObjectLike":
-        this.type = "objectLike";
-        this.parsedObj = obj.$ObjectLike;
-        break;
-      case "$ArrayWith":
-        this.type = "arrayWith";
-        this.parsedObj = obj.$ArrayWith;
-        break;
-      case "$Exact":
-        this.type = "exact";
-        this.parsedObj = obj.$Exact;
-        break;
-      case "$StringLike":
-        this.type = "stringLikeRegexp";
-        this.parsedObj = obj.$StringLike;
-        break;
-      default:
-        this.type = "exact";
-        this.parsedObj = obj;
-    }
+    this.parsedObj = {
+      matcher: obj
+    };
```

Comment:
The changes in this file don't look related to the nodejs upgrades, and I'm too lazy to figure out what this is. Can you explain what this is / why its necessary?

Reply:
Well looks like all of these changes come from the `integ-tests` module which has had lots of changes lately, so i'm not going to pay it any mind.